spec/support/revision_shared_examples.rb

```diff
@@ -61,6 +61,11 @@
 
       expect do
         visit project_trash_path(current_project)
+
+        within '#trash' do
```

Comment:
unrelated?

Reply:
Sort of? But this change is needed because this example expects the instance to only have 1 of the item in the trash. If for some reason there are other items in the trash, the spec fails. In the case of the liquid shared examples, it creates multiple test records that break this example.